pkg/backupdriver/backup_driver_controller_base.go

```diff
@@ -387,16 +385,11 @@ func (ctrl *backupDriverController) syncSnapshotByKey(key string) error {
 
 	// For guest clusters, add supervisor Snapshot to guest snapshot mapping if not already added
 	// The key is <supervisor snapshot CR name> and value is <guest snapshot namespace>:<guest snapshot CR name>
```

Comment:
Update comment

Reply:
The comment still holds true. We are adding the svcSnapshotCR name as the key with the value being the guestNS:guestSnapshotCR